### DISCUSSION TITLE
Black?

### DISCUSSION THREAD

**Discussion by j-bennet** [dbcli/mycli#1046] [Ideas]
Dear @dbcli/mycli-core developers,

I suggest we stop using `pep8radius` and switch to `black`. Black is slowly becoming a default tool to enforce code style. It's being maintained by PSF and constantly updated. `pep8radius` didn't have a release since 2015.

What do you think? Any 👍  or 👎 ?

Cheers.
Irina

**Comment by soumendrak** [comment]
+1 for Black

**Comment by rolandwalker** [comment]
Since we have now switched to `ruff format` this question is resolved!
